packages/react-sketch-canvas/src/Canvas/index.tsx

```diff
@@ -94,10 +101,20 @@ export const Canvas = React.forwardRef<CanvasRef, CanvasProps>((props, ref) => {
       if (!boundingArea) {
         return { x: 0, y: 0 };
       }
+      let x = 0;
+      let y = 0;
+
+      if (pointerEvent instanceof TouchEvent) {
+        x = pointerEvent.touches[0].pageX;
+        y = pointerEvent.touches[0].pageY;
+      } else {
+        x = pointerEvent.pageX;
+        y = pointerEvent.pageY;
+      }
```

Comment:
wouldn't be better if it would be like this?:
```js
let { pageX: x, pageY: y } = pointerEvent;
if(pointerEvent instanceof TouchEvent) {
  x = pointerEvent.touches[0].pageX;
  y = pointerEvent.touches[0].pageY;
}
```

Reply:
It will get ts error `Property 'pageX' does not exist on type 'TouchEvent | MouseEvent'.ts(2339)`;
The union type must be determined before it is used